packages/types/src/lib/interfaces.ts

```diff
@@ -300,6 +293,31 @@ export interface JsonSignChainDataRequest {
   exp: number;
 }
 
+export interface JsonSignSessionKeyRequestV1 {
+  sessionKey: string;
+  authMethods: AuthMethod[];
+  pkpPublicKey?: string;
+  authSig?: AuthSig;
+  siweMessage: string;
+  curveType: 'BLS' | 'ECDSA';
```

Comment:
I thought we had a `LIT_CURVE` introduced as a new enum could we it here?

Reply:
yea that's in the `constants` package, it will unfortunately cause circular dependencies. I think `constants` and `types` should just merge, they are static values anyway and don't depend on any other packages, but they love each other and we keep separating them 💔

Reply:
this makes sense to me, Will make a task as a follow up.